akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Multiplexer.scala

```diff
@@ -348,7 +348,7 @@ private[http2] trait Http2MultiplexerSupport { logic: GraphStageLogic with Stage
       }
 
       /** Pulled and data is pending but no connection-level window available */
-      case class WaitingForConnectionWindow(sendableOutstreams: immutable.Set[Int]) extends WithSendableOutStreams {
+      private[http2] case class WaitingForConnectionWindow(sendableOutstreams: immutable.Set[Int]) extends WithSendableOutStreams {
```

Comment:
To help the Scala compiler notice that nobody will see this class outside `http2`. It complained about:
> method pushControlFrame in class WaitingForConnectionWindow references private[http2] trait FrameEvent.